src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length calendar--start-day">
```

Comment:
these modifiers should be flexible and show calendar changes depends on these modifiers

Example:
calendar--month-length--30, calendar--month-length--29 etc
calendar--start-day--mon, calendar--start-day--wed etc

If you need any help feel free to ask at fe_chat